### ISSUE TITLE
E-mail link doesn't work on Accessibility page

### ISSUE THREAD

**Issue by basst85** [minvws/nl-covid19-notification-app-ios#56]
### Describe the bug, issue or concern

The e-mail link (helpdesk@coronamelder.nl) doesn't work on the accessibility page.
Nothing happens when you click on the e-mail address, only on long press.

### To Reproduce
1. Go to 'Vragen over de app?'
2. Click on 'Toegankelijkheid'
3. Click on 'helpdesk@coronamelder.nl'
Nothing happens, only on long press.

### Expected behavior
Open default e-mail client when you click on the e-mail address.

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: iOS 14.1

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

**Comment by roelspruit** [comment]
Thanks for the report. We're picking this up immediately. I made an internal ticket for this (ticket nr. 58919)

**Comment by roelspruit** [comment]
Hi. This issue has been fixed in the latest release of the app (1.0.12). The actual email address that is used is actually also incorrect but we will fix that outside this issue. Again, thanks for your report.
